Cargo.toml

```diff
@@ -214,72 +257,10 @@ tracing-log = "0.2"
 tracing-subscriber = { version = "0.3", features = ["env-filter", "json"] }
 tree_hash = "0.9"
 tree_hash_derive = "0.9"
-url = "2"
-uuid = { version = "0.8", features = ["serde", "v4"] }
-warp = { version = "0.3.7", default-features = false, features = ["tls"] }
-zeroize = { version = "1", features = ["zeroize_derive", "serde"] }
-zip = "0.6"
-
-# Local crates.
```

Comment:
The local crates have ben moved above and sorted together with the external crates. It looks fine to me, but let me know if you have any concerns.

Reply:
The separation is nice, but doesn't bother me too much. I think it's fine - if anyone has concerns we can revisit.